CHANGELOG.md

```diff
@@ -1,13 +1,19 @@
-## vUNRELEASED
+## v1.1.0
+
+This release is the first incremental release since 1.0. It includes bugfixes and some UX improvements.
 
 #### New features and UX changes
 
 - Add support for non-numerical UID/GID as specified in the appc spec ([#2159](https://github.com/coreos/rkt/pull/2159)).
```

Comment:
Nevermind, I guess I missed that PR: https://github.com/coreos/rkt/blob/master/stage1/init/common/pod.go#L282